src/components/MdList/MdListItem/MdListItem.vue

```diff
@@ -127,6 +135,9 @@
     text-transform: none;
 
     &:not(.md-list-item-default):not([disabled]) {
+      -webkit-user-select: none;
```

Comment:
Didnt autoprefix do this?

Reply:
Yep. This can be removed.

Reply:
@marcosmoura In dev it doesnt work without prefixes, i think we have broken autoprefixer?